lib/backbeat/Metrics.js

```diff
@@ -181,10 +190,10 @@ class Metrics {
                 completions: {
                     description: 'Number of completed replication operations ' +
                         '(count) and number of bytes transferred (size) in ' +
-                        `the last ${Math.floor(timeDisplay)} seconds`,
+                        `the last ${Math.floor(uptime)} seconds`,
```

Comment:
I think `Math.floor` here will still leave it at 1, `uptime` will be guaranteed at least 1

Reply:
Wow, apparently I don't know how `Math.floor` works! It does, however, look like `uptime` could be a decimal < 1. It's definitely a minor case though, since probably no requests will come in before one second has elapsed and it only affects the description 😸.